.markdownlint.yaml

```diff
@@ -1,7 +1,9 @@
 # All rules: https://github.com/DavidAnson/markdownlint#rules--aliases
-default: true  # Default state for all rules
-MD013: false  # line-length: We don't fill paragaraphs/limit line length
-MD034: false  # no-bare-urls - We use pymdownx.magiclink which allows bare urls
-MD046: false  # code-block-style - This doesn't play well with material's admonitions)
-MD024: false  # no-duplicate-heading - We use duplicate headings in the changelog.
-MD033: false  # no-inline-html - Too strict.
+default: true # Default state for all rules
+MD013: false # line-length: We don't fill paragaraphs/limit line length
+MD034: false # no-bare-urls - We use pymdownx.magiclink which allows bare urls
+MD046: false # code-block-style - This doesn't play well with material's admonitions)
+MD024: false # no-duplicate-heading - We use duplicate headings in the changelog.
+MD033: false # no-inline-html - Too strict.
+MD007:
+  indent: 4
```

Comment:
@marioevz I committed this here, but then cherry-picked it and made a separate PR that has already made it to main (#1460), if you rebase to main, this will change will disappear and all the indentation warnings here will (still) be fixed.